src/test/java/jenkins/plugins/git/AbstractGitSCMSourceTest.java

```diff
@@ -161,17 +161,17 @@ public void retrieveHeadsSupportsTagDiscovery_findTagsWithTagDiscoveryTrait() th
         sampleRepo.init();
         sampleRepo.git("checkout", "-b", "dev");
         sampleRepo.write("file", "modified");
-        sampleRepo.git("commit", "--all", "--message=dev-commit-message");
+        sampleRepo.git("commit", "--all", "--message=dev-commit-message", "--no-verify");
         long beforeLightweightTag = System.currentTimeMillis();
```

Comment:
this should also not use `currentTimeMillis` but `nonos` as the clock can change during testing causing flaky tests - but well that can be left for another day :)